app/js/views/reminders/reminder-item.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'components/react';
+import moment from 'components/moment';
+
+const COLOURS = ['red', 'orange', 'green', 'blue', 'violet'];
+
+export default class ReminderItem extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.reminder = props.reminder;
+  }
+
+  getColour(name) {
+    const hash = (string) => {
+      let hash = 0, i, chr, len;
+      if (string.length === 0) {
+        return 0;
+      }
+      for (i = 0, len = string.length; i < len; i++) {
+        chr = string.charCodeAt(i);
+        hash = ((hash << 5) - hash) + chr;
+        hash |= 0; // Convert to 32bit integer
+      }
+      return hash;
+    };
+
+    return COLOURS[hash(name) % COLOURS.length];
+  }
+
+  render() {
+    const reminder = this.reminder;
+    const contentClassName = [
+      'reminders__item-content',
+      this.getColour(reminder.recipient),
+    ]
+      .join(' ');
+
+    return (
+      <li className="reminders__item">
+        <div className="reminders__item-time">
+          <div>{moment(reminder.datetime).format('LT')}</div>
```

Comment:
Yeah there's the same problem with the time. The only way of fixing it is to use a table for the layout. Do you have a better idea?